engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/JavaInteropTest.scala

```diff
@@ -32,5 +32,57 @@ class JavaInteropTest extends InterpreterTest {
           |""".stripMargin
       eval(code) shouldEqual 21
     }
+
+    "allow importing static inner class and call methods on them" in {
```

Comment:
A test in Java, like `PrintTest` would be closer to my heart.

Reply:
The `JavaInteropTest.scala` had already existed. Shall I migrate it to Java then? It should not be difficult, these are just a few tests.

Reply:
Migrated to Java in e9dad1ddfa7fa1d2c5bc03c1d835623b243875e2

Reply:
Thank you, Pavel. It feels so much _warmer_ and _welcoming_!